XenkoShaderExplorer/MainViewModel.cs

```diff
@@ -64,7 +65,16 @@ public MainViewModel()
         {
             try
             {
-                RootShaders = BuildShaderTree().ToList();
+				_path = Directory
+					.EnumerateDirectories(BASE_PATH)
+					.First(s => 
+						System.IO.Path
+							.GetFileName(s)
+							.ToUpper()
+							.StartsWith("XENKO")
+					);
```

Comment:
It automatically figures out the correct path.
(It takes the first path whose folder name starts with "Xenko")